### ISSUE TITLE
Publish latest version to npm

### ISSUE THREAD

**Issue by AuspeXeu** [alphacharlie/node-ads1x15#10]
The version in the npm repository is still 1.0.0

**Comment by michalvankodev** [comment]
This actually solves issue #8 IS anyone able to republish?

**Comment by alphacharlie** [comment]
Yes. Sorry for the delay, Time issues...  I'll fire up a raspi and update the npmjs.com module this weekend sometime.
